app/controllers/katello/api/v2/alternate_content_sources_controller.rb

```diff
@@ -88,14 +87,14 @@ def update
       else
         find_smart_proxies
       end
-
```

Comment:
Try to avoid having random "style" changes in a PR.

Reply:
Good to know, thanks!